### ISSUE TITLE
FFMPEG broken a few versions back.  Video playback broken on my Intel with VAAPI

### ISSUE THREAD

**Issue by hobybrenner** [chrisbenincasa/tunarr#990]
I was hoping a new version would fix my issue, but so far I am seeing the same results.   Everything was working fine, same system plays Plex without issue as it has for past few years.

Both VAAPI and Intel Quicksync fail

i5 Intel system
Intel Corporation CoffeeLake-S GT2 [UHD Graphics 630]

docker-compose
```
<html>
<body>
<!--StartFragment-->
version: '3.8'
--
  | services:
  | tunarr:
  | image: chrisbenincasa/tunarr:latest-vaapi
  | container_name: tunarr
  | restart: unless-stopped
  | user: $USER:$GID
  | devices:
  | - /dev/dri:/dev/dri
  | ports:
  | - 8000:8000
  | environment:
  | - TZ=America/Chicago
  | - LOG_LEVEL=${TUNARR_LOG_LEVEL:-INFO}
  | volumes:
  | - /home/radmin/DATA/AppData/tunarr/config:/config/tunarr

<!--EndFragment-->
</body>
</html>
```
Error as follows...
```
2024-11-23T06:54:34.776Z [error]: Error playing video {"code":234,"cmd":"/usr/bin/ffmpeg -hide_banner -threads 4 -fflags +genpts+discardcorrupt+igndts -loglevel warning -hwaccel vaapi -vaapi_device /dev/dri/renderD128 -reconnect 1 -reconnect_on_network_error 1 -reconnect_streamed 1 -multiple_requests 1 -readrate 1.0 -readrate_initial_burst 45 -ss 2188.794 -i https://192-168-101-229.a8db068a673345e28f0e18f2bbba6876.plex.direct:32400/library/parts/97466/1540564425/file.mp4?X-Plex-Token=REDACTED -filter_complex [0:0]null[video];[video]format=nv12|vaapi,hwupload=extra_hw_frames=64[hwupload];[hwupload]scale_vaapi=w=1920:h=1080:mode=fast:extra_hw_frames=64[scaled];[scaled]hwdownload[hwdownloaded];[hwdownloaded]setsar=1[siz];[siz]format=nv12,hwupload=extra_hw_frames=64[hwuploaded];[0:1]anull[audio];[audio]aresample=48000,aresample=async=1:first_pts=0,apad=whole_dur=1209005ms[padded] -map [hwuploaded] -c:v h264_vaapi -b:v 10000k -maxrate:v 10000k -bufsize:v 1000k -sc_threshold 0 -video_track_timescale 90000 -map [padded] -flags cgop+ilme -b:a 192k -maxrate:a 192k -bufsize:a 50k -ac 2 -ar 48k -map_metadata -1 -movflags +faststart -muxdelay 0 -muxpreload 0 -fps_mode cfr -metadata service_provider=\"tunarr\" -metadata service_name=\"Holiday Commercials\" -t 1209005ms -g 119.88 -keyint_min 119.88 -force_key_frames expr:gte(t,n_forced*2) -f hls -hls_time 4 -hls_list_size 0 -segment_list_flags +live -hls_flags program_date_time+omit_endlist+discont_start+append_list -hls_segment_type mpegts -hls_base_url /stream/channels/569b4848-ef47-4176-a421-4ea8a3f7665d/hls/ -hls_segment_filename streams/stream_569b4848-ef47-4176-a421-4ea8a3f7665d/data%06d.ts -master_pl_name master.m3u8 streams/stream_569b4848-ef47-4176-a421-4ea8a3f7665d/stream.m3u8"}
```

**Comment by chrisbenincasa** [comment]
Hey - thanks for the report. However there are no error details in the log. Can enable both debug logging and ffmpeg logging and retry?

**Comment by hobybrenner** [comment]
Ok, I backed down to 15.10 which I thought was the last working version for me, and was still seeing issues.

I verified that no gpu works fine, just seeing 600 percent cpu usage with FFMpeg as expected... so I stopped all containers (including Plex) and fired things back up again, and now I am seeing better results.  This machine was rebooted, so I can't explain the behaviour here.

I am not sure if its the fact my Intel chipset is mapped as /dev/dri/card0 which I can verify by using intel-gpu-top or a combo of container restarts, but things are working with VAAPi just fine.  I will close this out.

Thanks for all your hard work!  I am using this to create holiday themed move and commercial channels, and loving all the updates!
